scripts/jenkins/push_docker.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# This script is present on workers but may not be present in a development
+# environment.
+
+if [ ${WORKERS+x} ]  # We are on a CI worker
```

Comment:
I'm not familiar with this env variable, where is it coming from?

Reply:
Ugh. This is what happens when I don't get enough sleep. This is supposed to be `WORKSPACE`. I'll fix.